src/connection/latestDriverVersionResolver.js

```diff
@@ -9,16 +9,49 @@ function LatestDriverVersionResolver(config, instanceConfig, logger) {
 
 LatestDriverVersionResolver.prototype.getLatestVersion = function (binary) {
   var that = this;
-  return that._getLatestMajorVersion(binary)
+  
+  if (binary.latestVersionUrlRedirect) {
```

Comment:
unnecesary 'if'?
where is latestVersionUrlRedirect set?

Reply:
if removed